index.html

```diff
@@ -204,31 +221,41 @@ <h3>Skills</h3>
       </section>
 
       <footer class="contact">
-        <h2>
-          I'm always interested in hearing about new projects, so if you'd like to chat please get
-          in touch.
-        </h2>
+        <div class="contact-form">
+          <h2>
+            I'm always interested in hearing about new projects, so if you'd like to chat please get
+            in touch.
+          </h2>
 
-        <form action="https://formspree.io/f/xlezyjjy" method="post" class="form-action">
-          <input type="text" name="name" placeholder="Full name" maxlength="30" required />
-          <input type="email" name="email" id="email" placeholder="Email address" required />
-          <textarea
-            name="message"
-            placeholder="Enter text here"
-            id="message"
-            cols="30"
-            rows="10"
-            maxlength="500"
-            required
-          ></textarea>
+          <form action="https://formspree.io/f/xlezyjjy" method="post" class="form-action">
+            <input type="text" name="name" placeholder="Full name" maxlength="30" required />
+            <input type="email" name="email" id="email" placeholder="Email address" required />
+            <textarea
+              name="message"
+              placeholder="Enter text here"
+              id="message"
+              cols="30"
+              rows="10"
+              maxlength="500"
+              required
+            ></textarea>
 
-          <button type="submit">Get in touch</button>
-        </form>
+            <button type="submit">Get in touch</button>
+          </form>
+        </div>
+        <div class="design-images3">
+          <img class="foot1" src="assets/images/foot.svg" alt="" />
+          <img class="foot2" src="assets/images/foot4.svg" alt="" />
+          <img class="foot3" src="assets/images/foot3.svg" alt="" />
+        </div>
 
         <div class="footer">
           <div class="design-images">
             <img class="illus3" src="assets/images/illustration-contact-form.svg" alt="" />
           </div>
+          <div class="line">
+            <hr>
+          </div>
```

Comment:
- [ ] Great job adding a grey line on top of the social media icon links in the Desktop version 💯👌, but you should remove it in the mobile version, kindly fix it 🤝

![4](https://user-images.githubusercontent.com/45231276/177730789-a92c343e-ca6b-4abb-aab5-08db37082f2d.PNG)

Reply:
Thank you. I will do that